amr-wind/wind_energy/ABLStats.H

```diff
@@ -64,6 +64,9 @@ public:
         m_abl_forcing = forcing;
     }
 
+    //! Calculate sfs stress averages
+    void calc_sfs_stress_avgs();
+
```

Comment:
Why public?

Reply:
Tried addressing in https://github.com/Exawind/amr-wind/pull/290/commits/22cbdfe875f7c910a3fd1369c871a361c06ac0dd